src/syntax/simple-statements.spec.ts

```diff
@@ -247,6 +247,26 @@ describe('Simple statements', () => {
     })
 
 
```

Comment:
From what I understand, in both `SET TRANSACTION` as well as `SET SESSION CHARACTERISTICS [...]` the `transaction_mode`s can be separated with a comma. I think there was also one in your initial example? 🤔 

If true, this version should be added to the tests.

Reply:
yup